docs/component.md

```diff
@@ -268,6 +271,32 @@ MyButton.register();
 <my-button>
 ```
 
+Note that these converters are also available should you wish to extend them in your own custom
+converter. You can import and use them like so:
+
+```javascript
+import Component, { toBoolean } from '@biotope/element';
+
+class MyButton extends Component {
+  ...
+}
+
+...
+MyButton.attributes = [
+  {
+    name: 'primary',
+    converter: (prop) => {
+      const parsedProp = toBoolean(prop);
+      console.log('original prop:', prop, ', parsed prop:', parsedProp);
```

Comment:
I don't know if we need the `console.log` here. Or you might add another example which is super short for _real_ usage. Like:

```
converter: toBoolean
```

Reply:
Fixed :)